README.md

```diff
@@ -1,84 +1,57 @@
-Zonemaster CLI
-==============
-[![Build Status](https://travis-ci.org/zonemaster/zonemaster-engine.svg?branch=master)](https://travis-ci.org/zonemaster/zonemaster-engine)
-[![CPAN version](https://badge.fury.io/pl/Zonemaster-CLI.svg)](https://metacpan.org/pod/Zonemaster::CLI)
+# Zonemaster-CLI
 
-### Purpose
 
-This Git repository is one of the components of the Zonemaster software and contains the source for the Zonemaster CLI utility.
+## Purpose
+
+This Git repository is one of the components of the Zonemaster software and
+contains the source for the Zonemaster-CLI utility.
 
 For an overview of the Zonemaster software, please see the
 [Zonemaster repository].
 
+
 ## Prerequisite
 
-Before you install the Zonemaster CLI utility, you need the
-Zonemaster Engine test framework installed. Please see the
+Before you install the Zonemaster-CLI utility, you need the Zonemaster-Engine
+test framework installed. Please see the
 [Zonemaster Engine installation instructions]
 
-Installation
-============
-
-Installation instructions for the CLI
-[installation] document.
-
-
-### Configuration 
-
-This repository does not need any specific configuration.
-
 
-### Docker
+## Installation
 
-To build a local image for Zonemaster CLI you need a [local Zonemaster Engine
-base image].
+For installation, see the [installation] document.
 
-Build a new local base image:
 
-```sh
-make all dist docker-build
-```
+## Configuration
 
-Tag the local base image with the current version number:
+This repository does not need any specific configuration.
 
-```sh
-make docker-tag-version
-```
 
-Tag the local base image as the latest version:
+## Docker
 
-```sh
-make docker-tag-latest
-```
+Zonemaster-CLI is available on [Docker Hub], and can be run without any
+installation. See [USING] Zonemaster-CLI for how to run it on Docker.
 
-Test a zone using the local base image:
+To build your own Docker image, see [Create Docker Image] documentation.
```

Comment:
Could this piece of documentation have a noun phrase for its name instead of a verb phrase? E.g. "Docker Image Creation".